thirdeye/thirdeye-pinot/src/test/java/org/apache/pinot/thirdeye/anomaly/detection/trigger/DataAvailabilityTaskSchedulerTest.java

```diff
@@ -110,8 +110,8 @@ public void testCreateMultipleTasks() {
     long detection2 = createDetection(2, metrics1, oneDayAgo, 0);
     List<Long> singleMetric = Collections.singletonList(metricId2);
     long detection3 = createDetection(3, singleMetric, oneDayAgo, 0);
-    createDataset(1, TEST_TIME);
-    createDataset(2, TEST_TIME);
+    createDataset(1, TEST_TIME, TEST_TIME);
```

Comment:
Why two TEST_TIME. Normally the two times should be different.

Reply:
Yes, they are always different in production. I simplify it in test case to make them the same, since we never compare those two timestamps anyway.